includes/admin/views/html-admin-page-addons.php

```diff
@@ -12,7 +12,7 @@
 }
 
 ?>
-<div class="wrap woocommerce wc_addons_wrap">
+<div class="wrap woocommerce">
```

Comment:
What is the impact of this change + removing the associated CSS rules? Screenshot before/after?

Reply:
This change has no big changes. Quite similar to #205 as snapshots below.

> ### Screenshot - before:
> ![Screenshot 2020-01-26 at 18 43 26](https://user-images.githubusercontent.com/7713923/73137940-58494b00-406e-11ea-9f1f-e019fc6e623c.png)
> ### Screenshot - after:
> 
> ![Screenshot 2020-01-26 at 18 43 42](https://user-images.githubusercontent.com/7713923/73137948-6a2aee00-406e-11ea-85f3-e84768cf011b.png)

Reply:
The rule `wc_addons_wrap` has a width of 1220px wrap around the text. This does not have any effect on the free flowing content that occupies the maximum space.